appveyor.yml

```diff
@@ -5,6 +5,9 @@ version: 1.0.{build}-appveyor
 
 image: Visual Studio 2017
 
+before_build:
```

Comment:
We can probably change this from `before_build:` to `install:`. 

From [appveyor.yml documentation](https://www.appveyor.com/docs/appveyor-yml/):
```
# scripts that run after cloning repository
install:
  # by default, all script lines are interpreted as batch
  - echo This is batch
  # to run script as a PowerShell command prepend it with ps:
  - ps: Write-Host 'This is PowerShell'
  # batch commands start from cmd:
  - cmd: echo This is batch again
  - cmd: set MY_VAR=12345
```